distribution/docs/src/main/resources/_contents/_running/monitoring-contents.adoc

```diff
@@ -110,7 +110,14 @@ Monitor system logs with the *LogViewer*, a convenient "viewing portal" for inco
 
 * Navigate to the LogViewer at \${secure_url}/admin/logviewer
 
+or
+
+* Navigate to the ${admin-console}
+* Navigate to the System tab and select Logs
+
 The LogViewer displays the most recent 500 log messages by default, but will grow to a maximum of 5000 messages.
+In order to view the logs, select the PAUSED button to see the current log messages in the LogViewer.
+Selecting the LIVE button will pause live logs from appearing in the LogViewer, this only affects the logs displayed by the LogViewer and does not affect the underlying log.
```

Comment:
Since the LIVE/PAUSED switch is a status indicator that displays the current state, it might be more straightforward to refer to it as a toggle button instead of a selectable button. ex, "... To view incoming logs, press the button in the top-left that reads "PAUSED" to toggle it to "LIVE" mode.  Switching this back to "PAUSED" will  prevent any new logs from being displayed in the LogViewer ..."